TheWarWithin/DeathKnightFrost.lua

```diff
@@ -438,16 +412,59 @@ spec:RegisterAuras( {
     -- https://wowhead.com/beta/spell=152279
     breath_of_sindragosa = {
         id = 152279,
-        duration = 10,
+        duration = 3600,
+        tick_time = 1,
+        max_stack = 1,
+        --[[alias = { "breath_of_sindragosa", " breath_of_sindragosa_real" },
+        aliasMode = "first",
+        aliasType = "buff",--]]
+    },
+    --[[breath_of_sindragosa = {
+        id = 152279,
+        duration = function () return floor( runic_power.current / 17 ) end,
         tick_time = 1,
         max_stack = 1,
         meta = {
-            remains = function( t )
-                if not t.up then return 0 end
-                return ( runic_power.current + ( runes.current * 10 ) ) / 17
```

Comment:
Was this not accurate?  I'm curious.  At very least, there should be a `remains` metafunction here that returns your `...breath_remains` expression.  Probably an `expires = function( t ) return query_time + breath_remains end`.

(I forgot what you called your expression and I'm only scrolling in one direction.)

Reply:
The new one accounts for the fact that you can't spend 6 runes at once and involves your GCD timer and 2-rune chunks to try and get ticks.